Source/FullScreenPokemon.ts

```diff
@@ -1013,14 +1013,20 @@ module FullScreenPokemon {
          * @todo Extend the use for any registered item, not just the bicycle.
          */
         keyDownSelect(thing: ICharacter, event?: Event): void {
+            var item: string = thing.FSP.ItemsHolder.getItem("SelectItem");
+
+            if (!item) {
+                thing.FSP.displayMessage(thing, "You can register an item to the SELECT key");
+            }
+
             thing.FSP.ModAttacher.fireEvent("onKeyDownSelect");
 
             if (thing.FSP.MenuGrapher.getActiveMenu()) {
                 return;
             }
 
-            if (!thing.FSP.toggleCycling(<IPlayer>thing)) {
-                thing.FSP.displayMessage(thing, thing.FSP.MathDecider.getConstant("items").Bicycle.error);
+            if (!thing.FSP.MathDecider.getConstant("items")[item].bagActivate(thing)) {
```

Comment:
To make this a little cleaner, store the [item] info in a separate `itemSchema` variable (or something named like that). That way you're not looking it up twice.